### DISCUSSION TITLE
Extend markdown syntax

### DISCUSSION THREAD

**Discussion by sadden3194** [usememos/memos#2113] [Ideas]
Hi all!

I would like to create tables.
something like this:

| Header 1  | Header 2  |
|-----------|-----------|
| Content 1 | Content 2 |

Documentation: https://www.tablesgenerator.com/markdown_tables
Best way I would be prefere is a WYSIWYG editor like https://www.tiny.cloud/

Could we realize it?

@boojack

**Comment by boojack** [comment]
You can get our supported markdown syntax in https://www.usememos.com/docs/getting-started/content-syntax
